seed/api/v2_1/views.py

```diff
@@ -117,8 +123,131 @@ def get_queryset(self):
     def building_sync(self, request, pk):
         """
         Return BuildingSync representation of the property
+
         ---
+        parameters:
+            - name: pk
+              description: The PropertyView to return the BuildingSync file
+              type: path
+              required: true
+            - name: organization_id
+              type: integer
+              required: true
+              paramType: query
+            - name: cycle_id
+              type: integer
+              required: true
+              paramType: query
+        """
+        # organization_id = request.data['organization_id']
+        cycle_id = request.query_params.get('cycle_id', None)
+
+        if not cycle_id:
+            return JsonResponse({
+                'success': False,
+                'message': "Cycle ID is not defined"
+            })
+        else:
+            cycle = Cycle.objects.get(pk=cycle_id)
+
+        try:
+            # TODO: not checking organization? Is that right?
```

Comment:
this is inheriting `SEEDOrgReadOnlyModelViewSet` which verifies the organization. So yes, this is okay.